docs/zkapps/how-to-deploy-a-zkapp.mdx

```diff
@@ -46,28 +48,34 @@ You see the following output:
 $ zk config
 
 Add a new network:
-✔ Choose a name (can be anything): · berkeley
-✔ Set URL to deploy to: · https://proxy.berkeley.minaexplorer.com/graphql
+✔ Create a name (can be anything): · berkeley
+✔ Choose the target network: · Testnet
+✔ Set the Mina GraphQL API URL to deploy to: · https://proxy.berkeley.minaexplorer.com/graphql
 ✔ Set transaction fee to use when deploying (in MINA): · 0.1
-✔ Create key pair at keys/berkeley.json
-✔ Add network to config.json
+✔ Choose an account to pay transaction fees: · Use stored account MyFeePayer (public key: B62...)
+✔ Use stored fee payer MyFeePayer (public key: B62...)
+✔ Create zkApp key pair at keys/berkeley.json
+✔ Add deploy alias to config.json
 
 Success!
 
 Next steps:
 
-- If this is a testnet, request tMINA at:
-  https://faucet.minaprotocol.com/?address=<YOUR-ADDRESS>
-- To deploy, run: `zk deploy berkeley`
+- If this is the testnet, request tMINA at:
+  https://faucet.minaprotocol.com/?address=<FEE-PAYER-ACCOUNT>
+- To deploy zkApp, run: `zk deploy berkeley`
 ```
 
-## Request funds from the faucet
+## Request funds from the Faucet
 
-To deploy your zkApp, you must have funds to pay for transaction fees.
+To deploy your zkApp, your fee payer account must have funds to pay for transaction fees.
 
-To get funds on the Berkeley Testnet, use the URL that was shown from the zkApp CLI output. Visit `https://faucet.minaprotocol.com/?address=<YOUR-ADDRESS>` and click **Request**.
+To get funds on the Berkeley Testnet, use the URL that was shown in the zkApp CLI output:
```

Comment:
Should this be `devnet`?

Reply:
Will be done as part of the work on https://github.com/o1-labs/docs2/issues/916.